ansible/roles/mysql/files/load_dump.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+mysql -uroot -p"$MYSQL_ROOT_PASSWORD" imovies < /docker-entrypoint-initdb.d/imovies_users.dump
```

Comment:
Do we really need this script. If I read the documentation correctly we can just put the dump directly in the mounted folder without an import script.

> Furthermore, it will execute files with extensions .sh, .sql and .sql.gz that are found in /docker-entrypoint-initdb.d